lib/internal/test_runner/test.js

```diff
@@ -264,15 +245,18 @@ class TestContext {
         };
       });
 
-      // This is a hack. It allows the innerOk function to collect the stacktrace from the correct starting point.
-      function ok(...args) {
-        if (plan !== null) {
-          plan.actual++;
+      if (!map.has('ok')) {
+        // This is a hack. It allows the innerOk function to collect the
+        // stacktrace from the correct starting point.
+        function ok(...args) {
+          if (plan !== null) {
+            plan.actual++;
+          }
```

Comment:
nit

```suggestion
          plan?.actual++;
```

Reply:
I'm going to leave this as is due to the following error. It also matches the existing style a few lines up in the same function.

```
node:internal/test_runner/test:252
          plan?.actual++;
          ^^^^^^^^^^^^

SyntaxError: Invalid left-hand side expression in postfix operation
```